lib/ae_page_objects/core/dsl.rb

```diff
@@ -20,7 +20,9 @@ def element(name, options = {}, &block)
 
       self.element_attributes[name.to_sym] = klass
 
-      define_method name do |&block|
+      define_method name do |&block_for_define_method|
```

Comment:
The handling in `visit` should be removed as well. That code is an artifact of when this gem was private and we were transitioning from one style to another. The `raise` should have been removed before open sourcing the gem.

In your example, the following is valid and the block is handled and defines a [nested element](https://github.com/appfolio/ae_page_objects/tree/130b710b85fcc95e2e2d03fe2f95feb21331ebcd#nested-elements):
```ruby
element :some_element do
  element :nested
end
```

>Since in Ruby you can pass a block to any method, that doesn't make sense to raise an exception for every method that does not do anything with the block.

So, you agree?